cli/cmd/get.go

```diff
@@ -157,6 +174,13 @@ func getAPIsInAllEnvironments() (string, error) {
 	var allTrafficSplitters []schema.TrafficSplitter
 	var allTrafficSplitterEnvs []string
 
+	type getAPIsOutput struct {
+		APIs  schema.GetAPIsResponse
+		Error string
+	}
+
+	allAPIsOutput := map[string]getAPIsOutput{}
```

Comment:
nit: add an inline comment, e.g.

```go
allAPIsOutput := map[string]getAPIsOutput{} // env name -> api output
```

Reply:
edit: I think it might be better to instead add `Env string` to the `getAPIsOutput` struct, and to serialize `[]getAPIsOutput` instead of the map. lmk if you want to discuss

Reply:
Great Idea